tests/bgp/test_bgp_suppress_fib.py

```diff
@@ -304,6 +304,23 @@ def setup_vrf_cfg(duthost, cfg_facts, nbrhosts, tbinfo):
     """
     cfg_t1 = deepcopy(cfg_facts)
     cfg_t1.pop('config_port_indices', None)
+    for loopback in cfg_t1['LOOPBACK_INTERFACE']:
+        loopback_items = loopback.split('|')
+        if len(loopback_items) == 2 and loopback_items[0] == 'Loopback0':
+            ipaddr = ipaddress.ip_address(loopback_items[1].split('/')[0])
+            if isinstance(ipaddr, ipaddress.IPv4Address):
+                router_id = str(ipaddr)
+                break
+    dut_asn = tbinfo['topo']['properties']['configuration_properties']['common']['dut_asn']
+    if 'BGP_GLOBALS' not in cfg_t1:
+        cfg_t1['BGP_GLOBALS'] = {}
```

Comment:
line 316 assigned empty, and the line 317 to check existence, should we remove line 317?

Reply:
Ack